source/features/shared/footer/components/Footer.scss

```diff
@@ -152,7 +152,7 @@
               text-decoration: none;
 
               &.emailLink {
-                color: rgba(235, 34, 86, 0.5);
+                color: rgba(31, 193, 195, 0.3);
```

Comment:
@DeeJayElly could you change this approach to CSS styling? This way you can use SCSS variables, that will be used for the different network versions.

Reply:
Fixed